### PR TITLE
Hover collection card on explore page

### LINKED ISSUE #5110: Hover card on explore
Collection hover card is missing on explore page 
image
- I think for smoother use there should be small delay from hover to displaying the hover card? wdyt

### PR DESCRIPTION
**Pull request by Jarsen136**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).

👇 \_ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring

## Context

- [x] Closes #5110
- [ ] Requires deployment <>

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted a screenshot of demonstrated change in this PR

#### Optional

- [ ] I've tested it at </bsx/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Had issue bounty label?

- [x] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=Caiv9TbPz68q5dC8EcHu5xKYPRnremimGzqmEejDFNpWWLG)

#### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

image

### CODE REVIEW

**Comment by exezbcz** [comment]
works fine, only the animation when you dont hover over the collection name should be as fast as with normal hover card.

**Comment by Jarsen136** [comment]
> works fine, only the animation when you dont hover over the collection name should be as fast as with normal hover card.


Kapture 2023-02-25 at 11 46 16
Do you mean the hover animation on gallery card is still too slow? I remember we have introduced the same animation with the same speed on all types of card. If it's still too slow, I could use a smaller delay for all of them.

ref: https://github.com/kodadot/nft-gallery/pull/4549

**Comment by exezbcz** [comment]
@Jarsen136 nope, that is not the case 

step one: 
hover over collection 
image
- the hover card pop ups - its delayed, that is good

step two: 
hover from the collection
- the hovercard disappears with delay as well and its slow - the animation (disappear) should be as fast as on normal cards 

does that makes sense? Thank you!

**Comment by Jarsen136** [comment]
> step two: hover from the collection
> 
> * the hovercard disappears with delay as well and its slow - the animation (disappear) should be as fast as on normal cards
> 
> does that makes sense? Thank you!

Yup, that makes sense. Will make it soon.

**Comment by exezbcz** [comment]
Yup, I believe the delay works well as a test, and if it becomes annoying, we can modify it. I'd also check whether or not I was wrong.

**Review by vikiival — APPROVED** [review]

File: libs/ui/src/components/NeoNftCard/NeoNftCard.vue [hunk 1]
```diff
@@ -55,6 +60,7 @@ import CommonTokenMoney from '@/components/shared/CommonTokenMoney.vue'
 import type { NFT } from '@/components/rmrk/service/scheme'
 import { getChainNameByPrefix } from '@/utils/chain'
 import { NftCardVariant } from '@kodadot1/brick'
+import CollectionDetailsPopover from '@/components/collectionDetailsPopover/CollectionDetailsPopover'
```
Comment (vikiival) [inline libs/ui/src/components/NeoNftCard/NeoNftCard.vue:63]: Why this import? Seems unused

File: libs/ui/src/components/NeoNftCard/NeoNftCard.vue [hunk 2]
```diff
@@ -17,15 +17,20 @@
...
-        <nuxt-link
-          v-if="
-            variant !== 'minimal' && (nft.collection.name || nft.collection.id)
-          "
-          :title="nft.collectionName || nft.collection.name"
-          :to="`/${prefix}/collection/${nft.collection.id}`"
-          class="is-size-7 nft-info-collection-name">
-          {{ nft.collection.name || '--' }}
-        </nuxt-link>
+        <CollectionDetailsPopover :show-delay="500" :nft="nft">
```
Comment (vikiival) [inline libs/ui/src/components/NeoNftCard/NeoNftCard.vue:20]: Your delay is hardcoded here

Comment (Jarsen136) [reply libs/ui/src/components/NeoNftCard/NeoNftCard.vue:20]: updated

File: libs/ui/src/components/NeoNftCard/NeoNftCard.vue (hunk 1 above)
Comment (Jarsen136) [reply libs/ui/src/components/NeoNftCard/NeoNftCard.vue]: Yes, I have removed it.

File: libs/ui/src/components/NeoNftCard/NeoNftCard.vue (hunk 2 above)
Comment (vikiival) [reply libs/ui/src/components/NeoNftCard/NeoNftCard.vue:20]: Looks like still here

Comment (Jarsen136) [reply libs/ui/src/components/NeoNftCard/NeoNftCard.vue]: Oh, sorry. I have updated it now.

**Comment by vikiival** [comment]
pay 35 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $35 @ 38.88 USD/KSM ~ 0.9 $KSM
🧗 [Caiv9TbPz68q5dC8EcHu5xKYPRnremimGzqmEejDFNpWWLG](https://kusama.subscan.io/account/Caiv9TbPz68q5dC8EcHu5xKYPRnremimGzqmEejDFNpWWLG)
🔗 [0x0e264b0180409e5a49c8c78fc470de89ac23eb636de8fbc186c959ab25175cd4](https://kusama.subscan.io/extrinsic/0x0e264b0180409e5a49c8c78fc470de89ac23eb636de8fbc186c959ab25175cd4)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Comment by yangwao** [comment]
pay 40 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $40 @ 5 USD/DOT ~ 8 $DOT
🧗 [16SjUbGKSdjCdWTy3NNT3JxbRVGGqD4mwkHpc6BD9U2Rp29Z](https://polkadot.subscan.io/account/16SjUbGKSdjCdWTy3NNT3JxbRVGGqD4mwkHpc6BD9U2Rp29Z)
🔗 [0x83b6dcdb46be96d3c8f71fe95abec038865361dc52cfbc30ee3fde302ed777b1](https://polkadot.subscan.io/extrinsic/0x83b6dcdb46be96d3c8f71fe95abec038865361dc52cfbc30ee3fde302ed777b1)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**